nim/tests/test_routes.py

```diff
@@ -0,0 +1,110 @@
+import json
+from nim.app import app
+import pytest
+
+
+@pytest.fixture
+def client_app():
+    return app.test_client()
+
+
+def assert_new_game_success(passed_client, req):
+    """
+    Asserts post to /new is 200
+    & state is not empty
+    :param passed_client: test_client
+    :param req: json to post
```

Comment:
We're in Python land, you should be accepting a `dict` instead. Plus now you're forcing all the calls to this function to call `json.dumps()`, clearly a sign that it needs factorizing.